portal-kernel/test/unit/com/liferay/portal/kernel/util/SystemPropertiesTest.java

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.kernel.util;
+
+import com.liferay.portal.kernel.test.ReflectionTestUtil;
+
+import java.util.Map;
+import java.util.Properties;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * @author Jiaxu Wei
+ */
+public class SystemPropertiesTest {
+
+	@Before
+	public void setUp() {
+		Map<String, String> properties = ReflectionTestUtil.getFieldValue(
```

Comment:
@W2jx Let's update the key & update logic.

Create a prefix for the keys, and in each test case, every time you need a unique key, use the prefix and a random string created using `RandomTestUtil`.

Then, change `setUp` into `@After tearDown` and remove properties with the prefix in the `tearDown`.

Reply:
I'll fix it.